frigate/track/tracked_object.py

```diff
@@ -338,6 +347,24 @@ def update(self, current_frame_time: float, obj_data, has_valid_frame: bool):
         self.current_zones = current_zones
         return (thumb_update, significant_change, autotracker_update)
 
+    def should_update_attribute(self) -> bool:
+        """Decides if attributes should be checked."""
+        if not self.active:
+            return False
+
+        if self.obj_data["label"] == "person" and not self.requires_face_detection:
+            return False
+        elif self.obj_data["label"] == "car" and not self.requires_lpr_detection:
+            return False
+
+        now = datetime.datetime.now().timestamp()
+
+        if now + 0.5 > self.last_published:
```

Comment:
I'm a little concerned this may be a bit excessive for the yolov9 license plate detection model, especially for users who run it on the CPU. At least for LPR, I wonder if something like a certain position change percentage of the frame of the `car` could be a trigger rather than a time trigger. Maybe something similar to what I did for the path points?

Reply:
yeah good idea, I think that would improve other things too